config/application.rb

```diff
@@ -10,6 +11,7 @@ module Socialnetwork
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.1
+    config.active_storage.service = :local
```

Comment:
I'm not sure if you should have the configuration at the environment level,
Like some value for `development.rb` and other values for `production.rb` and `test.rb`

Eventually you can look at the `rails_api_base` as a guide.

Reply:
Yes, I had it configured by environments